src/main/kotlin/three/consulting/epoc/service/CustomerService.kt

```diff
@@ -30,38 +30,36 @@ class CustomerService(private val customerRepository: CustomerRepository) {
             return CustomerDTO(customerRepository.save(customer))
         } else {
             val exception = UnableToCreateCustomerException()
-            logger.error("Cannot create new customer", exception)
+            logger.error(exception) {"Failed creating a new customer" }
             throw exception
         }
     }
 
     fun updateCustomerForId(customerRequest: CustomerDTO): CustomerDTO {
-        logger.info("Updating customer")
+        logger.info {"Updating customer with id: ${customerRequest.id}"}
         if (customerRequest.id != null) {
-            logger.debug("Updating customer with id: ${customerRequest.id}")
             val customer = Customer(customerRequest)
             return CustomerDTO(customerRepository.save(customer))
         } else {
             val exception = UnableToUpdateCustomerException()
-            logger.error("Cannot update customer", exception)
+            logger.error(exception) {"Cannot update customer"}
             throw exception
         }
     }
 
     fun deleteCustomer(customerId: Long) {
         logger.info("Deleting customer")
         try {
-            logger.debug("Deleting customer with id: $customerId")
+            logger.info {"Deleting customer with id: $customerId"}
```

Comment:
Maybe the first `logger.info("Deleting customer")` is somewhat redundant if the one with `id` is logged on info level.

Reply:
Ah I missed that one. It wasn't updated to the correct format either. Removed.